corehq/sql_accessors/migrations/0055_set_form_modified_on.py

```diff
@@ -6,7 +6,6 @@
 from corehq.sql_db.operations import RawSQLMigration
 
 migrator = RawSQLMigration(('corehq', 'sql_accessors', 'sql_templates'), {
-    'FORM_STATE_DELETED': XFormInstance.DELETED,
```

Comment:
this will have to stay until the SQL template files which reference it are removed (since you'll be creating new copies of the functions initially)